publish-site.sh

```diff
@@ -56,6 +56,8 @@ mkdocs build --clean
 # Copy the generated scala docs inside the site folder. 
 mkdir -p site/apidocs
 cp -R build-artifacts/scala-2.11/docs/* site/apidocs/
+mkdir -p site/funcRefs
+cp -R spark/sql/site/* site/funcRefs/
```

Comment:
@lizygeogy please confirm as to where you need to have these SQL function docs generated in the site directory

Reply:
@sumwale @paresh-p11 
We need to have the SQL function docs in a folder named **sql_functions** within **snappydata>site** folder.

Reply:
done